frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCard/FeatureStrategyMenuCard.tsx

```diff
@@ -12,7 +12,8 @@ interface IFeatureStrategyMenuCardProps {
     projectId: string;
     featureId: string;
     environmentId: string;
-    strategy: IStrategy;
+    strategy: Pick<IStrategy, 'name' | 'displayName' | 'description'> &
```

Comment:
Is there a type alias that could give it a descriptive name why we pick those fields in particular?